api/rest/client/client.go

```diff
@@ -290,8 +293,14 @@ func (c *defaultClient) setupHTTPClient() error {
 	}
 
 	c.client = &http.Client{
-		Transport: c.transport,
-		Timeout:   c.config.Timeout,
+		Transport: &ochttp.Transport{
```

Comment:
Not sure I understand what you mean by pollution of the http requests.

Reply:
This causes ochttp to encode the tracing context as request headers in the http request. I do not want any modification of the http requests made by the client if tracing is not enabled. Right now, my understanding is that this will do that even if tracing info is never sampled or exported. Is this not the case? I have not checked.

Reply:
Ahh yes that is correct. Is that really an issue considering http requests from ctl are not somewhere we are exactly bandwidth constrained and the size of the extra headers is minimal?

Reply:
I don't know how much of a practical issue it is, but it's quite easy to disable it and it's better if disabled by default. So let's disable it.